.github/workflows/stale.yml

```diff
@@ -0,0 +1,21 @@
+name: 'Close stale issues'
+on:
+  schedule:
+    - cron: '00 19 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v3
+        with:
+          stale-issue-message: |
+            This issue is marked as stale because author feedback has been requested for 30 days with no response. Please respond within 14 days or this issue will be closed due to inactivity.
+          close-issue-message: |
+            This issue was closed due to inactivity. If you can still reproduce this bug, please comment with the requested information, detailed steps to reproduce the problem, or any other notes that might help in the investigation.
+          start-date: '2021-01-06T00:00:00Z'
+          days-before-stale: 30
+          days-before-close: 14
+          stale-issue-label: stale
+          any-of-labels: 'Needs: Author Feedback'
+          exempt-milestones: 'Backlog'
```

Comment:
We might eventually want to make backlog fair game. That would be the semantic equivalent of "closed: low priority" from AzDO.

Also, it would be nice to add other milestones. I'm wondering if, rather than having a 16.11 milestone, a 17.0 milestone, etc., if we should have "current release," "last minor release," "penultimate minor release," and "last major release." Advantages: we could set it here without needing to update this, and we wouldn't have to update milestones on issues we didn't get to but are important. Downside: some issues should be targeted for a specific release, and they would silently move forward instead of staying where they should be.